app/components/candidate_interface/course_choices_review_component.rb

```diff
@@ -69,6 +74,41 @@ def study_mode_row(course_choice)
       }
     end
 
+    def financial_support_row(course)
+      {
+        key: 'Financial support',
+        value: course.financial_support,
+      }
+    end
+
+    def qualification_row(course)
+      {
+        key: 'Qualification',
+        value: DisplayQualification.call(qualification: course.qualification),
+      }
```

Comment:
Should this service just be called in the sync provider from find service?

I'm not 100% sure how we want this so be persisted in the db.